lib/LedgerSMB/Admin/Command/user.pm

```diff
@@ -507,23 +513,23 @@ Employee manager
 
 Employee number
 
-=item is_manager
+=item is-manager
```

Comment:
Doesn't take an argument, but how do I know?

Reply:
Fixed in documentation